api/topic_test.go

```diff
@@ -233,6 +233,345 @@ func TestGetTopicPrivateHandler(t *testing.T) {
 	})
 }
 
+// NOTE: The data within the following four sets of data returning functions
+//       are interlinked with one another by the SubtopicsIds
+
+// ================= - 1 has subtopics & points to 2 & 3
+func dbTopicWithID1(state models.State) *models.TopicResponse {
+	return &models.TopicResponse{
+		ID: "1",
+		Next: &models.Topic{
+			ID:    "1",
+			State: state.String(),
+			Links: &models.TopicLinks{
+				Subtopics: &models.LinkObject{
+					HRef: "http://example.com/topics/1/subtopics",
+				},
+			},
+			SubtopicIds: []string{"2", "3"},
+		},
+		Current: &models.Topic{
+			ID:    "1",
+			State: state.String(),
+			Links: &models.TopicLinks{
+				Subtopics: &models.LinkObject{
+					HRef: "http://example.com/topics/1/subtopics",
+				},
+			},
+			SubtopicIds: []string{"2", "3"},
+		},
+	}
+}
+
+// DB model corresponding to a topic in the provided state, without any download variant
+func dbTopic1(state models.State) *models.TopicResponse {
```

Comment:
These functions appear to be doing nothing other than providing an alias? Consider inlining the underlying function call, and potentially renaming it to the name of this function.

Reply:
I did rather wonder as to the point of these functions as i copied them from dp-dataset-api.
Code now inlined.